package.json

```diff
@@ -27,5 +37,21 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "eslint-config-airbnb": "^17.1.1",
+    "eslint-plugin-babel": "^5.3.0",
+    "eslint-plugin-import": "^2.18.0",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-react": "^7.14.2",
+    "eslint-plugin-react-hooks": "^1.6.1",
+    "expect": "^24.9.0",
```

Comment:
`expect` seems a little outdated. Let's give [jest](https://jestjs.io/) a try🤤

Reply:
OK, I guess `CRA` is already included `jest`, I'll give it a try, thanks:)